solr/bin/solr

```diff
@@ -2086,14 +2086,14 @@ else
   REMOTE_JMX_OPTS=()
 fi
 
-# Enable java security manager (limiting filesystem access and other things)
-if [ "$SOLR_SECURITY_MANAGER_ENABLED" == "true" ]; then
+# Disable java security manager (allowing filesystem access and other things)
+if [ "${SOLR_SECURITY_MANAGER_ENABLED:-true}" == "false" ]; then
```

Comment:
as a matter of style/consistency i would reverse this so we are comparing with "true", and change the comment back to "Enable" from "Disable". I think it is easier on the brain to think about positives (not the negatives).

so something like this pseudocode:
```
# Enable java security manager (limiting filesystem access and other things)
if [ "$SOLR_SECURITY_MANAGER_ENABLED:-true" == "true" ]; then
  SECURITY_MANAGER_OPTS=('-Djava.security.manager' \
    "-Djava.security.policy=${SOLR_SERVER_DIR}/etc/security.policy" \
    "-Djava.security.properties=${SOLR_SERVER_DIR}/etc/security.properties" \
    '-Dsolr.internal.network.permission=*')
else
  SECURITY_MANAGER_OPTS=()
fi
```

Essentially the same as existing logic, we just add the `:-true` so that if the variable is not set by the user in any way, we default its value to true.